package.json

```diff
@@ -239,6 +239,7 @@
     "yargs": "^17.5.1"
   },
   "dependencies": {
+    "@bsull/augurs": "0.2.0",
```

Comment:
Not urgent but I just released v0.3.1 of this package which has some significant perf improvements for changepoint detection, probably worth updating. It should be compatible for the APIs you're using 👍